### PR TITLE
Refactor: Integration event contracts

### PR DESCRIPTION
**Pull request by kft**
## Description
In order to maintain cohesiveness and keep a clear overview of the codebase, protobuf contracts and mapper are moved to the same folder and the POCO class (Integration event) it self.

`EnergySupplierChanged` contract has been refactored to comply with protobuf style guide, and TransactionId has been removed.

A couple of contracts currently not used have been removed.

### CODE REVIEW

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/IntegrationEvents/EnergySupplierChange/EnergySupplierChangedIntegrationEvent.proto [hunk 1]
```diff
@@ -19,9 +19,9 @@ option csharp_namespace = "Energinet.DataHub.MarketRoles.IntegrationEventContrac
...
  * This message is sent out when an energy supplier has been changed.
  */
+
 message EnergySupplierChanged {
-  string MeteringPointId = 1; // Unique metering point identification
-  string EnergySupplierGLN = 2; // Unique Energy Supplier identifcation.
-  string EffectiveDate = 3; // Date which the change of supplier goes into effect.
-  string TransactionId = 4; // Unique transaction ID of the process. Needed to effectuate the correct accepted process.
-}
+  string gsrn_number = 1; // Unique metering point identification
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/IntegrationEvents/EnergySupplierChange/EnergySupplierChangedIntegrationEvent.proto:24]: This is wrong. It is metering point ID as we are following ENTSO-E for domain naming and roles, and CIM internally. Checked with Preben, you are welcome to double check with him.

https://eepublicdownloads.entsoe.eu/clean-documents/EDI/Library/HRM/Harmonised_Role_Model_2020-01.pdf

**Review by erduna — CHANGES_REQUESTED** [review]
refactor to gsrn in integration event is wrong, as we use entso-e and cim naming in integration. Please confirm with Preben if you'd like.

Also, I disagree with removing the contracts for cancellation and future supplier change, but if you want to reimplement them (while excluding transaction ID) in a few weeks, feel free :)

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/IntegrationEvents/EnergySupplierChange/EnergySupplierChangedIntegrationEvent.proto (hunk 1 above)
Comment (kft) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/IntegrationEvents/EnergySupplierChange/EnergySupplierChangedIntegrationEvent.proto:24]: This has nothing to do CIM naming at all. This is an integration event. And integration events are not expressed in CIM. CIM is for EDI concerns.

**Review by Korgath — APPROVED** [review]

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/IntegrationEvents/EnergySupplierChange/EnergySupplierChangedIntegrationEvent.proto (hunk 1 above)
Comment (erduna) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/IntegrationEvents/EnergySupplierChange/EnergySupplierChangedIntegrationEvent.proto:24]: Metering point ID is not a cim term. It is entso-e, which we have agreed to use. GSRN is a danish implementation of a MP used for electricity, and is not suitable as a name for an integration event in GEH.

Comment (erduna) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/Integration/IntegrationEvents/EnergySupplierChange/EnergySupplierChangedIntegrationEvent.proto:24]: Also, may I suggest moving the currently unused contracts to a WIP folder, while removing the transaction id. As we are 100% know, and it is documented within the market, that a supplier change can be cancelled as well as initiated for a future date :)

**Review by cmgrd — APPROVED** [review]

**Review by cmgrd — APPROVED** [review]

**Review by erduna — APPROVED** [review]

**Pull request merged**